inbm/cloudadapter-agent/cloudadapter/cloud/client/inbs_cloud_client.py

```diff
@@ -77,6 +78,13 @@ def get_client_id(self) -> Optional[str]:
 
         return self._client_id
 
+    def set_dispatcher_state(self, state) -> None:
+        """Set the dispatcher state: running or dead
+
+        @param state: State of dispatcher agent to be set
+        """
+        self._dispatcher_state = state
```

Comment:
this should have a lock to be thread safe. for convenience, we could have a set_dispatcher_state and get_dispatcher_state that are both guarded by the lock

Reply:
get_dispatcher_state method added and used in _handle_inbm_command_request method. Threading lock added as well.